Makefile

```diff
@@ -20,6 +20,9 @@ run:
 test:
 	go test -v -count=1 ./tests/
 
+tcltest:
```

Comment:
We shall update this target to `tcl-test`, otherwise it's conflicting target name with filename and giving error as below.
> make: `tcltest' is up to date.

Reply:
Changes. thanks